vector/src/main/res/layout/item_editable_square_avatar.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/memberProfileInfoContainer"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?riotx_background"
+    android:padding="16dp">
+
+    <!-- I cannot do what I want using layer-list, do it manually here-->
+    <FrameLayout
+        android:id="@+id/itemEditableAvatarImageContainer"
+        android:layout_width="80dp"
+        android:layout_height="80dp"
```

Comment:
this is not the same size than in `item_editable_avatar`, so no coherency here?

Reply:
yes, needed to be bigger :/

Reply:
ok